src/styles/blocks/page.scss

```diff
@@ -0,0 +1,16 @@
+.page {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
It's a good practice to set a default `font-size` and `line-height` in the root element. This way, you can use `rem` units throughout your CSS, making it easier to scale your design.